src/bin/pgcopydb/schema.c

```diff
@@ -4759,6 +4626,99 @@ getTableArray(void *ctx, PGresult *result)
 }
 
 
+/*
+ * calculatePartKeyMinMaxValue calculates the min and max values for the
+ * candidate partition key of the given table.
+ */
+static bool
+calculatePartKeyMinMaxValue(PGSQL *pgsql, SourceTable *table)
```

Comment:
Naming: this does not look like a calculus, rather just fetching information from the source database. What about `getPartKeyMinMaxValue` instead?

Reply:
[32b403d](https://github.com/dimitri/pgcopydb/pull/684/commits/32b403d8b537a81134e65583f6821e733997efbc)